pkg/onchain/website/handler.go

```diff
@@ -38,6 +58,11 @@ func MassaTLDInterceptor(req *interceptor.Interceptor) *interceptor.Interceptor
 	massaIndex := strings.Index(req.Request.Host, ".massa")
 
 	if massaIndex > 0 && !strings.HasPrefix(req.Request.Host, "my.massa") {
+		// If the request is not https, redirect to https.
+		if req.Request.TLS == nil {
```

Comment:
if you do that I think that Thyra won't work anymore for firefox on windows as HTTPS is not working for that configuration right now.

Reply:
Furthermore, if you want to handle most requests, you should move this interceptor to the top.